doc/src/tutorials/physics/mechanics/rotating_disc_block_friction.rst

```diff
@@ -0,0 +1,150 @@
+
+   >>> class SlidingPathway(me.PathwayBase):
+   ...
+   ...     def __init__(self, fixed_point, contact_point, frame):
+   ...         """A custom pathway that moves along a rotating disc.
+   ...
+   ...         We assume that two points, which represent a block and a disc respectively,
+   ...         lie in the same plane, and ``SlidingPathway`` ensures that the forces and velocities
+   ...         are correctly related in this plane.
+   ...
+   ...         Parameters
+   ...         ==========
+   ...         fixed_point : Point
+   ...             Point fixed w.r.t the provided reference frame; the point on the disc.
+   ...         contact_point : Point
+   ...             Point with a time-dependent position w.r.t the fixed point, where the
+   ...             actuator forces will be applied. This point is assumed to be fixed to
+   ...             another body; the point on the block.
+   ...         frame : ReferenceFrame
+   ...             Reference frame in which the ``fixed_point`` has a zero velocity. The pathway
+   ...             is only valid as long as two points lie within the same plane.
+   ...
+   ...         """
+   ...
+   ...         self.fixed_point = fixed_point
+   ...         self.contact_point = contact_point
+   ...         self.frame = frame
+   ...
+   ...     @property
+   ...     def length(self):
+   ...         return self.contact_point.pos_from(self.fixed_point).magnitude()
+   ...
+   ...     @property
+   ...     def extension_velocity(self):
+   ...         """Extension velocity of the pathway.
+   ...
+   ...         The extension velocity of the pathway is the magnitude of the velocity of
+   ...         the ``contact_point`` relative to the frame in which the ``fixed_point``
+   ...         is stationary.
+   ...
+   ...         """
+   ...
+   ...         return self.contact_point.vel(self.frame).magnitude()
+   ...
+   ...     def to_loads(self, force):
+   ...         """Loads in the correct format to be supplied to `KanesMethod`.
+   ...
+   ...         Forces and torques applied to the ``contact_point`` and ``fixed_point``
+   ...         based on the friction force.
+   ...
+   ...         """
+   ...
+   ...         F = me.dynamicsymbols('F')
+   ...         direction = -self.contact_point.vel(self.frame).normalize()
+   ...         force = F
+   ...         return [
+   ...             me.Force(self.contact_point, force * direction),
+   ...             me.Force(self.fixed_point, -force * direction),
+   ...             me.Torque(self.frame, me.cross(self.contact_point.pos_from(self.fixed_point), -force*direction)),
+   ...         ]
+
+   >>> block = me.Particle('block', Q, m)
+   >>> disc = me.Particle('disc', P, m)
+   >>> normal_force = m * g
+   >>> pathway = SlidingPathway(P, Q, N)
+   >>> friction = me.CoulombKineticFriction(mu_k, normal_force, pathway, v_s=v_s, sigma=sigma, mu_s=mu_k)
+   >>> loads = friction.to_loads() + pathway.to_loads(force=None)
```

Comment:
The `ForceActuator` uses the `Pathway.to_loads` method. You don't have to call it here.